### PR TITLE
Endpoints: update for asset path and env changes

### PR DESCRIPTION
**Pull request by rsethur**
# PR into Azure/azureml-v2-preview

## Checklist

I have:

- [ X] read and followed the contributing guidelines

## Changes
 - update for asset path and env changes
 - update cli version to 0.0.49

Fixes #

### CODE REVIEW

File: examples/endpoints/online/managed/canary-declarative-flow/3-create-green.yaml [hunk 1]
```diff
@@ -35,16 +37,18 @@ deployments:
...
       scoring_script: score.py
-    environment: 
-        name: model2env
-        version: 1
-        path: .
-        conda_file: file:../../model-2/environment/conda.yml
+    environment:   
+      name: env-model2
+      version: 1      
+      path: .
```
Comment (mx-iao) [inline examples/endpoints/online/managed/canary-declarative-flow/3-create-green.yaml:48]: I don't think `path` field is needed here?

File: examples/endpoints/online/managed/canary-declarative-flow/4-flight-green.yaml [hunk 2]
```diff
@@ -13,16 +13,18 @@ deployments:
...
       scoring_script: score.py
-    environment: 
-        name: model1env
-        version: 1
-        path: .
-        conda_file: file:../../model-1/environment/conda.yml
+    environment:
+      name: env-model1
+      version: 1        
+      path: .
```
Comment (mx-iao) [inline examples/endpoints/online/managed/canary-declarative-flow/4-flight-green.yaml:24]: same here & with the other example yaml files

File: examples/train/tensorflow/tf_env.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-name: tf-env
+name: tf-
```
Comment (mx-iao) [inline examples/train/tensorflow/tf_env.yml:1]: did you mean "tf-environment"

Comment (rsethur) [reply examples/train/tensorflow/tf_env.yml:1]: good catch - will fix

File: examples/endpoints/online/managed/canary-declarative-flow/4-flight-green.yaml (hunk 2 above)
Comment (rihorn2) [inline examples/endpoints/online/managed/canary-declarative-flow/4-flight-green.yaml:24]: path is optional here

File: examples/train/tensorflow/tf_env.yml (hunk 3 above)
Comment (rihorn2) [inline examples/train/tensorflow/tf_env.yml:1]: I believe you meant tf-environment, based on the above uses

**Review by rihorn2 — APPROVED** [review]

**Review by paulshealy1 — APPROVED** [review]

**Pull request merged**